main.cpp

```diff
@@ -1024,12 +1027,18 @@ static void spawn_watchman(void) {
 
 #ifdef USE_GIMLI
   spawn_via_gimli();
-#elif defined(__APPLE__)
-  spawn_via_launchd();
 #elif defined(_WIN32)
   spawn_win32();
 #else
-  daemonize();
+  if (no_system_spawner) {
```

Comment:
I'm leaning towards this getting moved up to line 1018 above; I feel that that is more in alignment with us eventually moving launchd support to be a system spawner, and avoid adding more options to tackle that later.  I also feel that it will be handy for folks that are using the site spawning functionality in some interesting situations.

It is late and I think I should sleep on this before giving you the runaround :-)

Reply:
ah, I see what you mean. how's this look?